packages/samples/react/src/components/handout/basic.tsx

```diff
@@ -31,16 +31,15 @@ import {
 	KolNav,
 	KolProgress,
 	KolSelect,
-	KolTable,
+	KolTableStateful,
```

Comment:
Bitte in der Sample-App alles auf `KolTable` belassen. Wir würden das dann mittels dem Migrationstool machen.

Reply:
Am besten gleich ein Ticket für das Migrationsskript anlegen.

Reply:
https://github.com/public-ui/kolibri/issues/6325